packages/css/badge.css

```diff
@@ -0,0 +1,105 @@
+.ds-badge {
+  --dsc-badge-background: var(--ds-color-accent-base-default);
+  --dsc-badge-color: var(--ds-color-accent-contrast-default);
+  --dsc-badge-padding: 0 var(--ds-spacing-1);
+  --dsc-badge-height: 14px;
+  --dsc-badge-width: 14px;
+
+  background: var(--dsc-badge-background);
```

Comment:
Suggest we use `background-color` to be consistent with CSS for our other components.